rules-reviewed/eap6/uncategorized/environment-dependent.windup.xml

```diff
@@ -23,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic class instantiation" effort="0" category-id="optional">
+                 <hint title="Dynamic class instantiation" effort="1" category-id="optional">
```

Comment:
This is an "optional" change. The real estimate for a migration is only focussing on the "mandatory" category. "Optional" efforts are relevant for a modernization/optimization/cleanup.

An intense usage of reflection and dynamic class instantiations "smells". Cleaning it up properly definitely will take a while. Here we have an indication that something could be done here.